doc/advanced-configuration/Cassandra.md

```diff
@@ -0,0 +1,70 @@
+## Advanced Cassandra configuration
```

Comment:
You need to add this file to `mkdocs.yml` in order to have it included on readthedocs.org.

Also please try following command localy from repo's root 
```bash
mkdocs serve
```

to verify if everything's rendered correctly.